### PR TITLE
Upgrade drupal core to version 10

### LINKED ISSUE #244: Upgrade to Drupal 10
- [x] Audit contributed modules active in site for compatibility with Drupal 10
- [x] Upgrade any that are to versions that are compatible with both Drupal 9 and 10
- [x] Add contrib project for admin theme seven (we had to do for researchdata https://github.com/pulibrary/researchdata/pull/258)
- [x] Any that are not compatible considering removing, otherwise create a new issue to deal with a blocker to the upgrade. 
- [x] Upgrade the .lando.yml file to use the drupal10 recipe. 
- [ ] Upgrade to Drupal 10. If it complains about deprecated modules, work on the tickets to remove those. If it doesn't complain, close the tickets.
   - [x] https://github.com/pulibrary/recap/issues/252 
   - [x] https://github.com/pulibrary/recap/issues/253
   - [x] https://github.com/pulibrary/recap/issues/254
- [x] Test on staging
- [x] Deploy to prod

This step-by-step guide was helpful to successfully upgrade the researchdata site: https://www.drupal.org/docs/upgrading-drupal/upgrading-from-drupal-8-or-later/upgrading-a-composer-based-site
See: https://www.drupal.org/docs/upgrading-drupal/upgrading-from-drupal-8-or-later/upgrading-from-drupal-9-to-drupal-10/overview

### PR DESCRIPTION
**Pull request by christinach**
closes #244

### CODE REVIEW

**Comment by christinach** [comment]
I found useful running:
`lando composer why-not drupal/core 10.0`
This one returned:
```
drupal/core-recommended  9.5.9         requires          drupal/core (9.5.9)                                      
drupal/health_check      1.3.0         requires          drupal/core (^8 || ^9)                                   
drupal/juicebox          3.0.0-alpha2  requires          drupal/core (^8 || ^9)                                   
drupal/libraries         3.0.0-beta2   requires          drupal/core (^8.8 || ^9)                                 
drupal/upgrade_status    3.14.0        requires          drupal/core (^8 || ^9)                                   
drupal/core              10.0.0        requires          symfony/console (^6.2)                                   
drupal/drupal            dev-main      does not require  symfony/console (but v4.4.49 is installed)               
drupal/core              10.0.0        requires          symfony/dependency-injection (^6.2)                      
drupal/drupal            dev-main      does not require  symfony/dependency-injection (but v4.4.49 is installed)  
drupal/core              10.0.0        requires          symfony/event-dispatcher (^6.2)                          
drupal/drupal            dev-main      does not require  symfony/event-dispatcher (but v4.4.44 is installed)      
drupal/core              10.0.0        requires          symfony/http-foundation (^6.2)                           
drupal/drupal            dev-main      does not require  symfony/http-foundation (but v4.4.49 is installed)       
drupal/core              10.0.0        requires          symfony/http-kernel (^6.2)                               
drupal/drupal            dev-main      does not require  symfony/http-kernel (but v4.4.50 is installed)           
drupal/core              10.0.0        requires          symfony/mime (^6.2)                                      
drupal/drupal            dev-main      does not require  symfony/mime (but v5.4.13 is installed)                  
drupal/core              10.0.0        requires          symfony/routing (^6.2)                                   
drupal/drupal            dev-main      does not require  symfony/routing (but v4.4.44 is installed)               
drupal/core              10.0.0        requires          symfony/serializer (^6.2)                                
drupal/drupal            dev-main      does not require  symfony/serializer (but v4.4.47 is installed)            
drupal/core              10.0.0        requires          symfony/validator (^6.2)                                 
drupal/drupal            dev-main      does not require  symfony/validator (but v4.4.48 is installed)             
drupal/core              10.0.0        requires          symfony/process (^6.2)                                   
drupal/drupal            dev-main      does not require  symfony/process (but v4.4.44 is installed)               
drupal/core              10.0.0        requires          symfony/yaml (^6.2)                                      
drupal/drupal            dev-main      does not require  symfony/yaml (but v4.4.45 is installed)                  
drupal/core              10.0.0        requires          twig/twig (^3.4.3)                                       
drupal/drupal            dev-main      does not require  twig/twig (but v2.15.5 is installed)                     
drupal/core              10.0.0        requires          guzzlehttp/guzzle (^7.5)                                 
drupal/drupal            dev-main      does not require  guzzlehttp/guzzle (but 6.5.8 is installed)               
drupal/core              10.0.0        requires          guzzlehttp/psr7 (^2.4)                                   
drupal/drupal            dev-main      does not require  guzzlehttp/psr7 (but 1.9.1 is installed)                 
drupal/core              10.0.0        requires          asm89/stack-cors (^2.1)                                  
drupal/drupal            dev-main      does not require  asm89/stack-cors (but 1.3.0 is installed)                
drupal/core              10.0.0        requires          psr/log (^3.0)                                           
drupal/drupal            dev-main      does not require  psr/log (but 1.1.4 is installed)  
```
I went ahead and ran for each module: `lando composer require --with-all-dependencies <name of the module>`. It could not persist the composer.lock changes but kept reverting them. 
Also this command:
`lando composer show --tree`

**Comment by christinach** [comment]
We're using 'drupal/juicebox' which seems to have a [test version for drupal 10](https://www.drupal.org/project/juicebox).

File: README.md [hunk 1]
```diff
@@ -1,190 +1,75 @@
...
-
-We utilize capistrano to deploy the code out to the server.  To deploy code to an existing server run
-`cap <server set> deploy ` (for example `cap production deploy`).
-
-To import a database run `cap <server set> drupal:database:import_dump SQL_DIR=<path to dump> SQL_FILE=<dump file name>`
-
-To install code on a blank sever you must deploy and upload a database, so you need to pass the database bump variables to the deploy command `cap <server set> deploy SQL_DIR=<path to dump> SQL_FILE=<dump file name>`
-
-To see a list of al available command run `cap -T`
+<img alt="Drupal Logo" src="https://www.drupal.org/files/Wordmark_blue_RGB.png" height="60px">
```
Comment (maxkadel) [inline README.md:1]: We should put this back to the previous readme

**Comment by christinach** [comment]
I attempted to uppgrade first to drupal 8. I ran successfully `lando composer update drupal --with-all-dependencies`.
When I run `lando drush updatedb` it fails: `In BootstrapHook.php line 32:
                                                                      
  Bootstrap failed. Run your command with -vvv for more information.  `. I stopped here.

**Comment by maxkadel** [comment]
Closed in favor of #262

**Pull request closed without merging**